sdk/freestanding-execution-engine/README.md

```diff
@@ -0,0 +1,36 @@
+# Goal of the exercise
+
+The purpose of this exercise is to get familiarised with how to load files from the host and into the Veracruz freestanding execution engine. The freestanding execution engine inside of the Veracruz instance allows a user's program to be executed.
```

Comment:
I'd rephrase this as something more direct, like "This document provides a walkthrough of how to load files...".

Also, the freestanding execution engine isn't inside a Veracruz instance: it's freestanding, to be used for debugging and development outside of a real Veracruz instance.

Reply:
Rephrased introduction.